src/platform/site-wide/announcements/config/index.js

```diff
@@ -24,7 +24,7 @@ const config = {
       showEverytime: true,
     },
     {
-      name: 'covid-vaccine-signup',
+      name: 'covid-19-vaccine-signup',
```

Comment:
Also need to revert this.

Reply:
Fixed this and the widgetTypes. I didn't catch WebStorm renaming these.